### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

**Review by lana-shanghai — APPROVED** [review]
LGTM

File: requirements/base.in [hunk 1]
```diff
@@ -1,3 +1,2 @@
-tox
 substrate-interface==0.13.9
--e git+https://github.com/gridsingularity/d3a-interface@master#egg=d3a_interface
+-e git+https://github.com/gridsingularity/d3a-interface@update_dependencies#egg=d3a_interface
```
Comment (Hassan754) [inline requirements/base.in:2]: reminder

File: requirements/base.txt [hunk 2]
```diff
@@ -1,129 +1,32 @@
...
-    # via substrate-interface
-
-# The following packages are considered to be unsafe in a requirements file:
-# setuptools
+-e git+https://github.com/gridsingularity/d3a-interface@update_dependencies#egg=d3a_interface  # via -r requirements/base.in
+attrs==21.2.0             # via d3a-interface, jsonschema
+base58==2.0.1             # via scalecodec, substrate-interface
+certifi==2021.10.8        # via d3a-interface, requests, substrate-interface
+chardet==4.0.0            # via d3a-interface, requests
+idna==2.10                # via d3a-interface, requests, substrate-interface
```
Comment (Hassan754) [inline requirements/base.txt:12]: You probably need to update this environment's pip tools, and recompile after reverting the d3a-interface branch

Comment (faizan2590) [reply requirements/base.txt]: tested with recreating environment and reinstallation of pip tools

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**